bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/component.jsx

```diff
@@ -165,13 +167,15 @@ class PresentationToolbar extends PureComponent {
     } = this.props;
 
     if (event) event.currentTarget.blur();
+    if (currentPoll()) makeCall('stopPoll');
```

Comment:
could we define stopPoll in the same context we define `nextSlide` and pick it from props?
This would mean  no need to import makeCall in the component